.github/workflows/build-pr.yml

```diff
@@ -114,7 +115,7 @@ jobs:
         run: |
           make e2e-deploy-gatekeeper GATEKEEPER_VERSION=${{ matrix.GATEKEEPER_VERSION }}
           make e2e-deploy-ratify GATEKEEPER_VERSION=${{ matrix.GATEKEEPER_VERSION }}
-          make test-e2e
+          make test-e2e GATEKEEPER_VERSION=${{ matrix.GATEKEEPER_VERSION }}
```

Comment:
curious why this was necessary?

Reply:
I made a change to test.bats which would run a target to deploy ratify with provided GK version.